estate/models/estate_property.py

```diff
@@ -0,0 +1,94 @@
+from datetime import timedelta
+from odoo import fields, models,api
```

Comment:
```suggestion
from odoo import fields, models, api
```
use space after comma `,` everywhere. and import must be in alphabetical order.